system-tests/README.md

```diff
@@ -1,122 +1,260 @@
+```bash
+./gradlew -DuseFsVault="true" :launcher:shadowJar
+```
 
-Publish Identity Hub libraries to local Maven artifactory by executing gradle command `./gradlew publishToMavenLocal` from Identity Hub root folder.
+From the `Registration Service` root folder, execute the following command:
 
-### Running test locally
+```bash
+./gradlew :launcher:shadowJar
+```
 
-MVD System tests can be executed locally against a local MVD instance. MVD runs three EDC Connectors and one Registration Service.
+From the `MVD` root folder execute the following commands to set the `Registration Launcher` path environment variable and start `MVD` using the `docker-compose.yml` file.
 
-First please make sure that you are able to build MVD locally as described in [Building MVD project](#building-mvd-project) section.
+```bash
+export REGISTRATION_SERVICE_LAUNCHER_PATH=/home/user/RegistrationService/launcher
+docker-compose -f system-tests/docker-compose.yml up --build
+```
 
-- We need to build EDC Connector launcher and Registration Service launcher.
-- Go to MVD root folder. And execute
+_Note for Windows PowerShell, the following commands should be used from the `MVD` project root.  (The path will depend on the location of the your `RegistrationService` project root):_
 
-    ```bash
-    ./gradlew -DuseFsVault="true" :launcher:shadowJar
-    ```
+```powershell
+$Env:REGISTRATION_SERVICE_LAUNCHER_PATH="c:/RegistrationService/launcher"
+docker-compose -f system-tests/docker-compose.yml up --build
+```
 
-- Go to Registration service root folder. And execute
+Once completed, following services will start within their docker containers:
+- 3 `EDC Connectors`
+  - _consumer-us_
+  - _consumer-eu_
+  - _provider_ (which will also be seeded with initial required data using a [postman collection](../deployment/data/MVD.postman_collection.json))
+- A `Registration Service`
+- A `HTTP Nginx Server` (to serve DIDs) 
+- An `Azurite` blob storage service 
+  
 
-    ```bash
-    ./gradlew :launcher:shadowJar
-    ```
+_Note, the `Newman` docker container will automatically stop after seeding initial data from postman scripts._
 
-- Start MVD using docker-compose.yml file.
+`EDC Connectors` need to be registered using `Registration Service` CLI client jar. After publishing `Registration Service` locally the client jar should be available under the `Registration Service` root project folder in _client-cli/build/libs_.
 
-    ```bash
-    export REGISTRATION_SERVICE_LAUNCHER_PATH=Registration service launcher path e.g. `/home/user/RegistrationService/launcher`.
-    docker-compose -f system-tests/docker-compose.yml up --build
-    ```
+```bash
+# Replace path according to your local set up
+export REGISTRATION_SERVICE_CLI_JAR_PATH=c:/RegistrationService/client-cli/build/libs/registration-service-cli.jar
 
-  for windows powershell
+# Register Participants
+./system-tests/resources/register-participants.sh
```

Comment:
note that @cpeeyush is changing this in https://github.com/agera-edc/MinimumViableDataspace/pull/178 (work in progress), will need to be updated